docs: warnings for dotenv vulnerability (#10138)
langflow-ai/langflow: docs/docs/Support/release-notes.mdx

Highlights of this release include the following changes.
For all changes, see the [Changelog](https://github.com/langflow-ai/langflow/releases).

> ### Known issue: .env file not loading {#env-file-bug}

Reviewer: ```suggestion
### Known issue, potential security vulnerability: .env file not loaded in versions 1.6.0 through 1.6.3 {#env-file-bug}
```

> Langflow versions `1.6.0` through `1.6.3` have a critical bug where environment variables from `.env` files are not being read. This affects all deployments using environment variables for configuration, including security settings.

Reviewer: ```suggestion
Langflow versions 1.6.0 through 1.6.3 have a critical bug where environment variables from `.env` files aren't read.
This affects all deployments using environment variables for configuration, including security settings.
```

> **Potential Security Impact:**

Reviewer: ```suggestion
:::Warning Potential security vulnerability
```

If you deployed with `AUTO_LOGIN=false` in your `.env` file, upgrading to these versions will cause Langflow to fall back to default settings, potentially giving all users superuser access.

> Database credentials, API keys, and other sensitive configuration may not be loaded from `.env` files.

Reviewer: ```suggestion
Additionally, database credentials, API keys, and other sensitive configurations can't be loaded from `.env` files.
```

> **DO NOT** upgrade to Langflow versions `1.6.0` through `1.6.3` if you use `.env` files for configuration. Instead, upgrade to `1.6.4`, which includes a fix for this bug.

Reviewer: ```suggestion
_Don't_ upgrade to any Langflow version from 1.6.0 through 1.6.3 if you use `.env` files for configuration.
Instead, upgrade to 1.6.4, which includes a fix for this bug.
:::
```
